src/sentry/services/hybrid_cloud/app/impl.py

```diff
@@ -18,15 +18,29 @@
     RpcSentryAppService,
     SentryAppInstallationFilterArgs,
 )
-from sentry.services.hybrid_cloud.filter_query import FilterQueryDatabaseImpl
+from sentry.services.hybrid_cloud.auth import AuthenticationContext
+from sentry.services.hybrid_cloud.filter_query import (
+    FilterQueryDatabaseImpl,
+    OpaqueSerializedResponse,
+)
+from sentry.services.hybrid_cloud.user import RpcUser
 
 
-class DatabaseBackedAppService(
-    FilterQueryDatabaseImpl[
-        SentryAppInstallation, SentryAppInstallationFilterArgs, RpcSentryAppInstallation, None
-    ],
-    AppService,
-):
+class DatabaseBackedAppService(AppService):
```

Comment:
Should this class also have a `_fq` cached property?

Reply:
a8b6f2e6ffa2968fc80a022bb59804132a7380ec